src/app/themes.css

```
@@ -7,7 +7,7 @@
 .deep-dark-high-contrast,
 .dark > .deep-dark-high-contrast {
   --app-font-sans: var(--font-inter), ui-sans-serif, system-ui, sans-serif;
-  --app-font-serif: var(--font-instrument-serif), serif;
+  --app-font-serif: ui-serif, Georgia, serif;
```

Stylelint (17.11.0)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)